count-stats/pom.xml

```diff
@@ -36,9 +36,15 @@
             <scope>test</scope>
         </dependency>
         <dependency>
-            <groupId>org.junit.jupiter</groupId>
+            <groupId>org.testcontainers</groupId>
+            <artifactId>postgresql</artifactId>
+            <version>1.15.3</version>
+            <scope>test</scope>
+        </dependency>
+        <dependency>
+            <groupId>org.testcontainers</groupId>
             <artifactId>junit-jupiter</artifactId>
```

Comment:
I think we should explicitly exclude the JUnit4 package that ships with Spring if we want to include this.

Reply:
Current version of Testcontainers depends on Junit-4. [See this issue](https://github.com/testcontainers/testcontainers-java/issues/970). 
They have a plan to remove this dependency in Testcontainers 2.0.
Till that time, we can explore other options like the one given [here](https://github.com/testcontainers/testcontainers-java/issues/970#issuecomment-625044008)